### ISSUE TITLE
Connection Error while trying download c4/en.realnewslike

### ISSUE THREAD

**Issue by nikregrado** [tensorflow/datasets#2187]
I using default script to download c4 with config en.realnewslike, and getting an error 
RuntimeError: requests.exceptions.ConnectionError: HTTPSConnectionPool(host='commoncrawl.s3.amazonaws.com', port=443): Max retries exceeded with url: /crawl-data/CC-MAIN-2019-18/segments/1555578530176.6/wet/CC-MAIN-20190421040427-20190421062427-00300.warc.wet.gz (Caused by NewConnectionError('<urllib3.connection.HTTPSConnection object at 0x7f1dcd61ba90>: Failed to establish a new connection: [Errno 110] Connection timed out')) [while running 'Map(download_url)']
After 10 error like this, job on dataflow failing, I use tesnorflow-datasets==3.1.0 and python 3.7

**Comment by Conchylicultor** [comment]
@adarob, do you know about this error ?

**Comment by adarob** [comment]
Sounds like it could be an transient issue with the Common Crawl download server being overloaded. Have you tried rerunning it?

**Comment by nikregrado** [comment]
Yes, I’m trying a week to download this dataset, and got this error)

**Comment by adarob** [comment]
Is it always that exact error or is it for different URLs?  You could also try using `--register_checksums` which will force it to download everything in one thread. It will be very slow but may get around this issue, which I suspect is throttling from the server.

**Comment by nikregrado** [comment]
This is not help, i'm got the same error but now on different urls
